config/spring19_seminar/cola_var/sample.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Move to scripts/spring19_seminar

Reply:
Done!

Reply:
Also relocated the configs under config/spring19_seminar (the cola_var folder is removed)